app/models/xccdf/benchmark.rb

```diff
@@ -22,7 +22,7 @@ class Benchmark < ApplicationRecord
 
     LATEST_SUPPORTED_VERSIONS = {
       'xccdf_org.ssgproject.content_benchmark_RHEL-6': '0.1.28',
-      'xccdf_org.ssgproject.content_benchmark_RHEL-7': '0.1.46',
+      'xccdf_org.ssgproject.content_benchmark_RHEL-7': '0.1.49',
```

Comment:
See the Jira. Sorry for the missing information.

```suggestion
      'xccdf_org.ssgproject.content_benchmark_RHEL-7': '0.1.50',
```